### PR TITLE
10567[integration-test-base refactor]: Initial code refactoring. 

### PR DESCRIPTION
**Pull request by abhioncbr**
First PR of code refactoring of integration-test-base 
- removing some of the dead code not getting used.
- removal of the arguments not getting used in the function.
- exception handling at a couple of places.
- minor logic change in the functionality.

cc: @Jackie-Jiang

### CODE REVIEW

File: pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java [hunk 1]
```diff
@@ -163,7 +163,7 @@ public static void setUpH2TableWithAvro(List<File> avroFiles, String tableName,
       params.append(",?");
     }
     PreparedStatement h2Statement =
-        h2Connection.prepareStatement("INSERT INTO " + tableName + " VALUES (" + params.toString() + ")");
+        h2Connection.prepareStatement("INSERT INTO " + tableName + " VALUES (" + params.toString() + ")");;
```
Comment (Jackie-Jiang) [inline pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java:166]: Revert

File: pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java [hunk 2]
```diff
@@ -903,25 +900,21 @@ private static void comparePinotResultsWithExpectedValues(Set<String> expectedVa
   }
 
   private static String removeTrailingZeroForNumber(String value, String type) {
+    String upperCaseType = StringUtils.toUpperCase(type);
```
Comment (Jackie-Jiang) [inline pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java:903]:
```suggestion
    String upperCaseType = type.toUpperCase();
```

File: pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java [hunk 3]
```diff
@@ -55,6 +55,7 @@
 import org.apache.avro.io.BinaryEncoder;
 import org.apache.avro.io.EncoderFactory;
 import org.apache.avro.util.Utf8;
+import org.apache.hadoop.util.StringUtils;
```
Comment (Jackie-Jiang) [inline pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java:58]: We don't tend to use this library

**Review by Jackie-Jiang — APPROVED** [review]
Thanks for the clean up!

File: pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java (hunk 3 above)
Comment (abhioncbr) [reply pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java:58]: My intent was to use `org.apache.commons.lang3.StringUtils;` but imported the wrong one :(

File: pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java (hunk 2 above)
Comment (abhioncbr) [reply pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java]: `type.toUpperCase()` is not null safe, though we aren't sending the `null` value.
For future perspective changed it to `String upperCaseType = StringUtils.upperCase(type);`

**Pull request merged**